### PR TITLE
Create Code of Conduct

### PR DESCRIPTION
**Pull request by gregor-bmks**
Add document containing code of conduct.

### CODE REVIEW

File: code-of-conduct.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# Code of Conduct
+
+## Introduction
+
+Biomakespace is building a multidisciplinary, volunteer-driven community laboratory for collaborative experimentation at the interface of biology and engineering. We aim to provide a supportive, safe environment for discovery and innovation through team-based projects and curiosity-driven research.
```
Comment (gregor-bmks) [inline code-of-conduct.md:5]: It may be good to elaborate more on topics of openness and equity.

File: code-of-conduct.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+## Scope
+
+This code of conduct applies to all Biomakespace interactions and spaces, both online and in person. By attending Biomakespace events, space and posting in our online forums, you are agreeing to this code of conduct.
+
```
Comment (gregor-bmks) [inline code-of-conduct.md:14]: Need to add section on revising the code of conduct.

File: code-of-conduct.md (hunk 1 above)
Comment (galen-bmks) [reply code-of-conduct.md:5]: 👍

File: code-of-conduct.md (hunk 2 above)
Comment (galen-bmks) [reply code-of-conduct.md:14]: 💡

File: code-of-conduct.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
+# Code of Conduct
+
+## Introduction
+
+Biomakespace is building a multidisciplinary, volunteer-driven community laboratory for collaborative experimentation at the interface of biology and engineering. We aim to provide a supportive, safe environment for discovery and innovation through team-based projects and curiosity-driven research.
+
+We strive to make our community open to everybody, regardless of scholarly or professional background, gender identity and expression, sexual orientation, ability, physical appearance, body size, race, age, economic background, country of origin or employment, religion, and other characteristics. As we come from different backgrounds, it is important to be intentional about providing respectful, equitable spaces — both online and in person — for our community to come together and engage in constructive, respectful discourse.
+
+We particularly strive for equity. We are intentional about actively reducing the inequitable barriers that stand between science and those who create, use, and lean from it.
```
Comment (galen-bmks) [inline code-of-conduct.md]: There is a typo: "lean" should be "leaRn".

**Review by galen-bmks — CHANGES_REQUESTED** [review]
There is a typo to be corrected, otherwise it looks fine.

**Review by galen-bmks — APPROVED** [review]
Looks good! 🏎 🥇 🏆

**Pull request merged**